analyze_benchmark_data/benchmark_analysis.md

```diff
@@ -0,0 +1,174 @@
+    ## 6                                                        There had to be some trigger that caused one to develop it and the other not,
+
+Here we get the `file_id`, the true speaker of the quote, their true gender, and the quote in question. Now lets find what we get out of coreNLP, which we will compare against
+
+``` r
+# 
+raw_quote_file = paste(proj_dir, 
+                    "/benchmark_data/benchmark_quote_table_raw.tsv", 
+                    sep="")
+
+raw_quote_df = read_corenlp_quote_files(raw_quote_file)
+
+head(raw_quote_df)
+```
+
+    ##                                                       file_id     est_speaker
+    ## 1                                          d41586-020-00889-6           Addex
+    ## 2                                          d41586-020-00889-6           Addex
+    ## 3                                          d41586-020-00889-6           Addex
+    ## 4 quantum-technology-probes-ultimate-limits-of-vision-1.17731    Angelo Bassi
+    ## 5 quantum-technology-probes-ultimate-limits-of-vision-1.17731 Anthony Leggett
+    ## 6    ebola-experience-leaves-world-no-less-vulnerable-1.18844      Ashish Jha
+    ##   est_gender canonical_speaker    partial_name
+    ## 1       MALE      spokesperson           Addex
+    ## 2       MALE       Neena Nizar           Addex
+    ## 3       MALE      spokesperson           Addex
+    ## 4       MALE           Unknown    Angelo Bassi
+    ## 5       MALE   Anthony Leggett Anthony Leggett
+    ## 6       MALE        Ashish Jha      Ashish Jha
+    ##                                                                                                          quote
+    ## 1                                                                           The COVID-19 situation is dynamic,
+    ## 2                I feel like we were chugging along on a train and then somebody dropped a huge boulder on it.
+    ## 3 We are now seeing impacts on clinical-trial continuity in all the regions where we conduct clinical studies.
+    ## 4                     something like a superposition of two different perceptions, even if just for an instant
+    ## 5                                               We don't know at what stage it's going to break down — or how.
+    ## 6                                      I honestly doubt that many of these recommendations will be acted upon.
+
+The main columns of interest are `est_gender` and `est_speaker`, which we will compare between lines that have the same `quote` and `file_id`.
+
+### analyzing quote data
+
+First, lets look at the *benchmark* data, to see if there exist any gender disparity evidence. <img src="benchmark_analysis_files/figure-markdown_github/unnamed-chunk-4-1.png" style="display: block; margin: auto;" />
+
+Ok, so we see some signal. Now what does it look like for our estimated gender?
+
+<img src="benchmark_analysis_files/figure-markdown_github/unnamed-chunk-5-1.png" style="display: block; margin: auto;" />
```

Comment:
Just to confirm you are predicting gender from the quotes alone or is it from speaker name based on the coreNLP package?

Reply:
I am predicting gender in one of two ways 1) coreNLP reports the gender 2) if coreNLP reports the name of the speaker and does not know the gender I use the package `genderizeR` in order to predict the gender using historical data. 
So it isn't from the quote, but instead the named entity associated with the quote.